README.md

```diff
@@ -1,20 +1,23 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
+(a) What is a click handler?
 
-(b) Where in your JS file should you put it, and WHY? 
+  # a click handler is a method that gets attached to a tag that you want to click.
 
-### Question 2
-If you get the error `$ is undefined`, what does that mean and what could you do to fix it? 
+(b) Where in your JS file should you put it, and WHY?
+  # It is recommened to be put at the end of the body tag. As for the why... for loading purposes but I am not very sure.
 
+### Question 2
+If you get the error `$ is undefined`, what does that mean and what could you do to fix it?
+  # It means that Jquery is not being referenced. To solve it just add the link to the Jquery library.
 ### Question 3
-What should go in the `$(document).ready()` part of your JS file? 
-
+What should go in the `$(document).ready()` part of your JS file?
+  # Whatever script of code that you want to run only until the page has finished loding or when the DOM is done being created... ?
```

Comment:
Yes, you're right. Specifically, click handlers should go in the `$(document).ready()`.